tools/integrations/manage.sh

```diff
@@ -69,6 +69,12 @@ cmd_publish() {
   echo "Publishing new version ($versioned_image)"
   docker push "$versioned_image"
   docker push "$latest_image"
+
+  # publish spec to cache. do so, by running get spec locally and then pushing it to gcs.
+  local tmp_spec_file; tmp_spec_file=$(mktemp)
+  docker run --rm "$versioned_image" spec | jq .spec > "$tmp_spec_file"
+  gcloud auth activate-service-account --key-file /Users/charles/Downloads/prod-ab-cloud-proj-bdb658ebbe5a.json
```

Comment:
we should inject this key in CI and pull from an env variable 
There generally shouldn't be a strong need to publish from a local machine as you can already skip tests from CI is needed

Reply:
kk. made it so it was possible to so locally if needed. happens by default in the publish command on GH